helm/router/templates/NOTES.txt

```diff
@@ -1,6 +1,6 @@
 1. Get the application URL by running these commands:
 {{- if .Values.ingress.enabled }}
-  http{{ if $.Values.ingress.tls }}s{{ end }}://{{ .Values.ingress.host | default (printf "www-origin.%s.%s.%s" .Release.Namespace .Values.govukEnvironment .Values.govukDomainExternal ) }}{{ .path }}
+  http{{ if $.Values.ingress.tls }}s{{ end }}://{{ .Values.ingress.host | default (printf "www-origin.%s.eks.%s.%s" .Release.Namespace .Values.govukEnvironment .Values.govukDomainExternal ) }}{{ .path }}
```

Comment:
Just a heads-up that this naming scheme won't work with TLS (unless/until we set up a k8s operator for issuing certs), because a single wildcard cert can only match one level unfortunately. I'll change it in my PR shortly.

Reply:
ok cool, we should decide how we are going to manage the domains, will we be able to deploy stuff in namespaces and map it accordingly in domains?

Reply:
I don't think there's anything much to decide at this point. As long as we're able to test stuff for the purposes of getting this project done, we're fine. We don't want to take on the huge task of replacing govuk-docker when we need to focus on the April 2022 deadline.

Reply:
Later on, we could potentially have an operator for issuing certs, which would enable deploying a complete Publishing stack into a separate namespace and still have TLS work.